dali/python/nvidia/dali/math.py

```diff
@@ -21,32 +21,161 @@ def _arithm_op(*args, **kwargs):
     setattr(sys.modules[__name__], "_arithm_op", nvidia.dali.ops._arithm_op)
     return nvidia.dali.ops._arithm_op(*args, **kwargs)
 
-def exp(value):
-    """Fills the output with exponential of value.
-    :rtype: TensorList of exp(value). If value is an integer, the result will be float, otherwise the type is preserved.
+
+def sqrt(input):
+    """Fills the output with square root of the input.
```

Comment:
General comment:
`fills the output` sounds like the output is somehow passed to this function.
```suggestion
    """Returns element-wise square root of `input`.
```
or
```suggestion
    """Computes element-wise square root of `input`.
```
or
```suggestion
    """Computes square root of values in `input`.
```

Reply:
Done, I was missing a good doc for that.